adhocteam/Head-Start-TTADP
# Add screen reader instructrions for continuing session

**Description of change**

 * When inactive the modal that asks the user to continue their session now includes a screen ready only instruction to reduce confusion
 * `REACT_APP_INACTIVE_MODAL_TIMEOUT` and `REACT_APP_SESSION_TIMEOUT` set to proper values. These environment variables can be set in circle ci at build. In the sandbox and dev environments they are currently set to 5 minutes and 10 minutes.

**How to test**

1. Login to sandbox https://tta-smarthub-sandbox.app.cloud.gov/
2. Wait 5 minutes, you should see the logout modal
3. Wait an additional 5 minutes, you should get logged out

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/79

**Checklist**

- [n/a] ~Documentation updated~
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
